index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css">
+  <title>Learn something</title>
+</head>
+
+<body>
+  <!-- HERO VID -->
+  <div class="video-container">
+    <video autoplay loop muted>
+        <source src="dog.mp4" type="video/mp4">
+        Your browser does not support the video tag.
+    </video>
+    <div class="overlay-text">
+      <h1>YO!</h1>
+    </div>
+ </div>
+
+  <!-- HERO IMAGE -->
+  <!-- <header>
+  <div class="hero-image">
+    <div class="hero-text">
+      <h1>Hello</h1>
+      <p>Wanna Learn Something?</p>
+    </div>
+  </header> -->
+
+  <section class="middlesection">
+    <!-- FORM -->
+  <div class="form-container">
+    <h2>SIGN UP</h2>
+  <form action="https://httpbin.org/anything" method="POST">
+    <label>
+  <input type="text" name="fullname" placeholder="Full Name" required>
+</label>
+
+<label>
+  <input type="email" name="email" placeholder="Email" required>
+</label>
+
+<label>
+  <input type="password" name="password" placeholder="Password" minlength="8" required>
+</label>
+
```

Comment:
Would like to see more work on the indention in the html file. That would make the code more easy to read!

Reply:
Thank you Erica, I will work on this :)